plugins/spi/SPIOutput.cpp

```diff
@@ -261,6 +297,10 @@ SPIOutput::SPIOutput(const UID &uid, SPIBackendInterface *backend,
                   "APA102 Pixel Brightness Combined",
                   sdc_irgb_combined));
 
+  personalities.insert(personalities.begin() + PERS_TLC5971_INDIVIDUAL - 1,
+    Personality(m_pixel_count * TLC5971_SLOTS_PER_DEVICE,
+      "TLC5971 Individual Control (16bit per channel)"));
```

Comment:
We probably ideally want both 8 and 16bit individual and combined options eventually, but feel free to fix the underlying issues first.

Reply:
yes - that is the plan when the rest works :-)
for the combined options there are more then one way to solve it...
1. repeat/copy one set of driver channels (24ch) to all others
1. repeat/copy the first 3 LED values (3 or 6 ch for 8 or 16bit modes) to all other positions

what do you think does make more sense?